test/unit/services/notification_delivery_service_test.rb

```diff
@@ -37,4 +37,16 @@ def test_invalid_event
       NotificationDeliveryService.new(notification).call
     end
   end
+
+  def test_false_boolean_attribute_is_valid_event
```

Comment:
`test_false_or_empty_attribute_is_valid_event`

That excludes `nil` but allow `''`

Reply:
> That excludes `nil` but allow `''`

This was widely discussed in https://github.com/3scale/system/pull/9439#discussion_r218047593 😄

Reply:
So here I am not testing the `empty` like `''` because it is neither good nor bad 🤷‍♀️

Reply:
hmm so we are not testing those case?
To me your change is stating that false is valid and empty too.
The only invalid is `nil`

Reply:
![image](https://user-images.githubusercontent.com/11318903/48907836-f50bc980-ee68-11e8-9f3b-ba13dfba0653.png)
![image](https://user-images.githubusercontent.com/11318903/48907842-fccb6e00-ee68-11e8-81eb-646de97841c4.png)
![image](https://user-images.githubusercontent.com/11318903/48907864-06ed6c80-ee69-11e8-916c-23e961d65b47.png)
![image](https://user-images.githubusercontent.com/11318903/48907887-1967a600-ee69-11e8-8d72-70696c4c1bde.png)

Reply:
![image](https://user-images.githubusercontent.com/11318903/48907986-5f246e80-ee69-11e8-9e39-9c38b160a9e0.png)

Reply:
As spoken in Slack after this, what we want is:
  - `nil` <- INVALID
  - `false` (instance of `FalseClass`, not any `falsy` value) <- VALID
  - `''` and `[]` we currently don't really mind, it is not relevant **at this moment**, and in any feature anyone may desire them to either be valid or invalid but we don't know yet, so for now we make them valid and we make the test as regression test or documentation as what it actually does independently of needed or not. And I leave this comment in the PR in case that anyone in the future wants to find out what was going on when this was done 😄